components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/GatewayArtifactsMgtDAO.java

```diff
@@ -517,6 +517,75 @@ public List<APIRuntimeArtifactDto> retrieveGatewayArtifactsByAPIIDs(List<String>
         return apiRuntimeArtifactDtoList;
     }
 
+    public List<APIRuntimeArtifactDto> retrieveGatewayArtifactsOnlyByAPIIDs(List<String> apiIds, String tenantDomain)
+            throws APIManagementException {
+        // Logging the API ID List
+        log.info("Getting runtime artifacts for the API ID List: " + apiIds.toString());
```

Comment:
Should be a debug log

Reply:
Fixed: https://github.com/wso2/carbon-apimgt/pull/11304/commits/c1dd6055755e9d451636e15a814c7aff415a99e1